src/components/HomepageFeatures.tsx

```diff
@@ -29,31 +29,29 @@ const FeatureList: FeatureItem[] = [
         Validators.
       </>
     ),
-    link: "/docs/dvl/intro",
+    link: "/next/learn/intro/launchpad",
```

Comment:
Not a now thing but after we release a version with this change maybe we remove the /next/ so we always point to the latest released version rather than the WIP version.

Reply:
Well we've removed the homepage now, so it shouldn't matter anyway

Reply:
I think the homepage still shows up when you click that link, since it opens the v1.1.2. If you click on "next" on the dropdown on the top right corner it immediately takes you to the first page of the docs...